sidebar_translations.json

```
@@ -67,10 +67,12 @@
   "develop.misc.saved_data": "Saved Data",
   "develop.misc.text_and_translations": "Text and Translations",
   "develop.misc.debugging": "Debugging Mods",
-  "develop.misc.migrating_mappings": "Migrating Mappings",
-  "develop.misc.migrating_mappings.loom": "Loom (Gradle)",
-  "develop.misc.migrating_mappings.ravel": "Ravel (IntelliJ IDEA)",
-  "develop.misc.automatic_testing": "Automated Testing",
+  "develop.porting": "Porting",
+  "develop.porting.introduction": "Introduction",
+  "develop.porting.migrating_mappings": "Migrating Mappings",
+  "develop.porting.migrating_mappings.loom": "Loom (Gradle)",
+  "develop.porting.migrating_mappings.ravel": "Ravel (IntelliJ IDEA)",
+  "develop.porting.automatic_testing": "Automated Testing",
```

```suggestion
  "develop.misc.automatic_testing": "Automated Testing",
```

actually, it needs to be moved up also